### DISCUSSION TITLE
How to received the RGB-pointcloud?

### DISCUSSION THREAD

**Discussion by lennard28** [ifm/ifm3d#385] [Q&A] [answered]
Hi,

I have an O3R222 camera connected to the OVP800. In the ifmVisionAssistant there is a rgb pointcloud ("overview"), which combines the rgb and 3d data.
I am able to receive the RGB and Depth Image separately by accessing the 2 different ports, but how can I retrieve the RGB pointcloud shown in the "overview" or how can I map the RGB data to the corresponding depth data?

![grafik](https://user-images.githubusercontent.com/47598150/217261615-647d06ad-ae30-4d6e-b529-02b942652880.png)

**Comment by lola-masson** [comment, marked as answer]
Hi @lennard28,
Currently the mapped 2D/3D image is not provided as part of the API. We are working on finalizing an example showing how to generate it using the camera's intrinsics. Email support.robotics@ifm.com and we will provide the example.
